ubuntu22.04/nvidia-driver

```diff
@@ -73,14 +76,10 @@ _install_prerequisites() (
     mkdir -p /lib/modules/${KERNEL_VERSION}/proc
 
     echo "Installing Linux kernel headers..."
-    apt-get -qq install --no-install-recommends linux-headers-${KERNEL_VERSION} > /dev/null
+    #apt-get -qq install --no-install-recommends linux-headers-${KERNEL_VERSION} > /dev/null
```

Comment:
Since re-using the kernel headers from the host is a separate feature, let's make this change in a follow-up PR.